Resources/doc/get_it_started.md

```diff
@@ -6,32 +6,29 @@ The preferred way to install the library is using [composer](http://getcomposer.
 Run composer require to add dependencies to _composer.json_:
 
 ```bash
-php composer.phar require "payum/payum-bundle" "payum/offline" "php-http/guzzle6-adapter"
+composer require "payum/payum-bundle" "payum/offline" "php-http/guzzle7-adapter"
 ```
 
 _**Note**: Where payum/offline is a php payum extension, you can for example change it to payum/paypal-express-checkout-nvp or payum/stripe. Look at [supported gateways](https://github.com/Payum/Core/blob/master/Resources/docs/supported-gateways.md) to find out what you can use._
 
 _**Note**: Use payum/payum if you want to install all gateways at once._
```

Comment:
This line can also be removed, since we discourage installing `payum/payum`. See https://github.com/Payum/Payum/issues/912